Explorer/Assets/DCL/PlacesAPIService/PlacesAPIClient.cs

```diff
@@ -85,8 +86,9 @@ public PlacesAPIClient(IWebRequestController webRequestController, IDecentraland
             GenericDownloadHandlerUtils.Adapter<GenericGetRequest, GenericGetArguments> result = webRequestController.GetAsync(
                 url, ct,
                 ReportCategory.UI,
-                signInfo: WebRequestSignInfo.NewFromUrl(url, timestamp, "get"),
-                headersInfo: new WebRequestHeadersInfo().WithSign(string.Empty, timestamp));
+                signInfo: signRequest ? WebRequestSignInfo.NewFromUrl(url, timestamp, "get") : null,

```

Comment:
This doesnt entirely fixes the problem as we do still call: https://github.com/decentraland/unity-explorer/blob/6b01212076f8f2fca61915c0172c6d5a819e2ca3/Explorer/Assets/DCL/MapRenderer/MapLayers/PointsOfInterest/ScenesOfInterestMarkersController.cs#L76
Which is a signed fetch that is throwing identity exceptions